lib/content-services/src/lib/search/search-query-builder.service.ts

```diff
@@ -21,7 +21,7 @@ import { SearchConfiguration } from './search-configuration.interface';
 import { BaseQueryBuilderService } from './base-query-builder.service';
 
 @Injectable({
-    providedIn: 'root'
+    providedIn: 'any'
```

Comment:
is that on purpose?

Reply:
yes it is on purpose. We need a different service instance between the global search and the content node selector otherwise what you search on one it will remain as search term on the other

Reply:
Just remove the parameters then, and leave 
```
@Injectable()
```